.spectral.yaml

```diff
@@ -1,6 +1,11 @@
-extends: '@rhoas/spectral-ruleset'
+extends: 'bin/rhoasapi-installation/node_modules/@rhoas/spectral-ruleset/ruleset.yaml'
 rules:
   no-$ref-siblings: off
   rhoas-response-media-type: off
   oas3-valid-schema-example: off
+  rhoas-external-$ref: off
```

Comment:
Why is this turned off?

Reply:
Updated some of the rules to give warnings instead of error out, the reasoning is because the openapi generator can now deal with external refs in OAS files.

Reply:
I might be misunderstanding something, if the rule is set to `off` won't it be ignored completely?

Reply:
yes it will.

Reply:
Thanks. Then I guess what we want is to always enable this so that we get the warning message we need. 
i.e the line `rhoas-external-$ref: off` will have to be removed.